frontend/src/pages/ActivityReport/Pages/goalsObjectives.js

```diff
@@ -89,6 +89,14 @@ Buttons.propTypes = {
 const GoalsObjectives = ({
   reportId,
 }) => {
+  // Check if this is the first time the user has opened the page,
+  // if so, we need to refresh the page to ensure mesh-kit doesn't have duplicated data.
+  const isFirstLoad = window.localStorage.getItem(`goals-${reportId}-loaded`) === null;
+  if (isFirstLoad) {
+    window.localStorage.setItem(`goals-${reportId}-loaded`, 'true');
+    window.location.reload();
+  }
+
```

Comment:
Maybe worth documenting that this is a brute-force temporary workaround for the MeshKit bug.